integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -4714,6 +4723,68 @@ msgstr "Automatisch den Titel des verlinkten Inhalts als Linktext verwenden"
 msgid "Media Library..."
 msgstr "Medienbibliothek..."
 
+#: cms/templates/ajax_poi_form/_poi_form_widget.html
+#: cms/templates/events/event_form.html cms/templates/imprint/imprint_form.html
+#: cms/templates/imprint/imprint_sbs.html cms/templates/pages/page_form.html
+#: cms/templates/pages/page_sbs.html cms/templates/pois/poi_form.html
+msgid "Publish"
+msgstr "Veröffentlichen"
+
+#: cms/templates/ajax_poi_form/poi_box.html
+msgid "Name of event location"
+msgstr "Name des Veranstaltungsortes"
+
+#: cms/templates/ajax_poi_form/poi_box.html
+msgid "Remove location from event"
+msgstr "Veranstaltungsort von Veranstaltung entfernen"
+
+#: cms/templates/ajax_poi_form/poi_box.html
+msgid ""
+"Create an event location or start typing the name of an existing location. "
+"Only published locations can be set as event venues"
+msgstr ""
+"Erstellen Sie einen Veranstaltungsort oder beginnen Sie den Namen eines "
+"bestehenden Veranstaltungsortes einzutippen. Nur öffentliche Orte können als "
+"Veranstaltungsorte verwendet werden"
+
+#: cms/templates/ajax_poi_form/poi_box.html
+#: cms/templates/pois/poi_form_sidebar/position_box.html
+msgid "Address"
+msgstr "Adresse"
+
+#: cms/templates/ajax_poi_form/poi_box.html
+msgid "Open on Google Maps"
+msgstr "Auf Google Maps öffnen"
+
+#: cms/templates/ajax_poi_form/poi_box.html
+#: cms/templates/pois/poi_form_sidebar/contact_box.html
+msgid "Contact details"
+msgstr "Kontaktdaten"
+
+#: cms/templates/ajax_poi_form/poi_box.html
+msgid "E-mail address: "
+msgstr "E-Mail-Adresse: "
+
+#: cms/templates/ajax_poi_form/poi_box.html
+msgid "Not provided"
+msgstr "Keine Angabe"
+
+#: cms/templates/ajax_poi_form/poi_box.html
+msgid "Phone number: "
+msgstr "Telefonnummer: "
+
+#: cms/templates/ajax_poi_form/poi_box.html
+msgid "Website: "
+msgstr "Website: "
+
+#: cms/templates/ajax_poi_form/poi_box.html
+msgid "The new location was successfully created."
+msgstr "Ein neuer Ort wurde erfolgreich erstellt"
```

Comment:
I see that it was just moved, but let's take the opportunity to be more consistent
```suggestion
#: cms/templates/ajax_poi_form/poi_box.html
msgid "The new location was successfully created."
msgstr "Ein neuer Ort wurde erfolgreich erstellt."
```